src/games/GameList.tsx

```diff
@@ -32,6 +33,7 @@ export const GameList = () => (
   <List filters={postFilters}>
     <Datagrid>
       <TextField source="id" />
+      <ShowButton />
```

Comment:
Weird placement for a ShowButton. You should place it at the end of the row IMO.